src/tic_tac_toe/ai.clj

```diff
@@ -3,33 +3,25 @@
             [tic-tac-toe.game :refer  [game-depth winner?]]
              [tic-tac-toe.protocol.player :refer [PlayerProtocol]]))
 
-(def marker-1 "x")
-(def marker-2 "o")
-(def place-holder (atom {:player-marker marker-1 :ai-marker marker-2}))
-
-(def max-score 1)
-(def min-score -1)
-(def draw-score 0)
-
-(defn switch-markers [place-holder marker-1 marker-2]
- (swap! place-holder assoc (first(keys @place-holder)) marker-2 (last(keys @place-holder)) marker-1))
- 
-(defn score-game [board]
-  (let [winner (winner? board)]
+(defn score-game [game]
+  (let [winner (winner? (game :board))]
```

Comment:
You should access items in maps using the keyword as the function. Please take a look at the Clojure style guide if you haven't already.
https://github.com/bbatsov/clojure-style-guide#keywords-as-fn-to-get-map-values